src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -239,14 +239,14 @@ def _assert_object_definitions_are_correct(
 
 def _find_existing_objects(
     object_type: ObjectType,
-    objects: List[Dict],
+    objects: List[FunctionSchema | ProcedureSchema],
```

Comment:
Maybe create a Type from this two objects?
It is repeated further down in code. 
WDYT?

Reply:
Added, good suggestion 👌